<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rscmgmt</class>
 <widget class="QWidget" name="rscmgmt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Los!</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditSearch"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Anzeige nach:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Suche nach</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="ResourceTree" name="treeWidget">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="description">
         <property name="text">
          <string>Beschreibung</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="type">
         <property name="text">
          <string>Typ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxSetFilter">
         <item>
          <property name="text">
           <string>Kategorie</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ort</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="make">
         <property name="text">
          <string>Hersteller</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="9" column="1">
         <widget class="QLabel" name="labelSerialNumber_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="labelSerialnumber">
          <property name="text">
           <string>Seriennummer</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QTextEdit" name="textEditDescription">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labelDescription">
          <property name="text">
           <string>Beschreibung</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="labelType_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelType">
          <property name="text">
           <string>Typ</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelMake">
          <property name="text">
           <string>Hersteller</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelCategory">
          <property name="text">
           <string>Kategorie</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelRoom">
          <property name="text">
           <string>Raum</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelIVS">
          <property name="text">
           <string>IVS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCostLocation">
          <property name="text">
           <string>Kostenstelle</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelActivationDate">
          <property name="text">
           <string>Aktivierungsdatum</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelEquipmentnumber">
          <property name="text">
           <string>Ausstattungsnummer</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="labelMake_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelCategory_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelRoom_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelIVS_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="dateEditActiveDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelEquipmentnumber_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelCostLocation_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="labelPurchasePrice">
          <property name="text">
           <string>Einkaufspreis</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="labelPurchasePrice_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="labelMinQty_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="labelMinQty">
          <property name="text">
           <string>Minimale Menge</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="labelQtyInStock_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="labelQtyInStock">
          <property name="text">
           <string>Lagernde Menge</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="labelTotalQty_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="labelTotalQty">
          <property name="text">
           <string>Gesamtmenge</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResourceTree</class>
   <extends>QTreeWidget</extends>
   <header>resourcetree.h</header>
   <slots>
    <signal>treeNeedsUpdate(int,int)</signal>
    <slot>updateTree()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>treeWidget</sender>
   <signal>treeNeedsUpdate(int,int)</signal>
   <receiver>rscmgmt</receiver>
   <slot>updateTree(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateTree(int,int)</slot>
 </slots>
</ui>
